app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -35,10 +37,12 @@ def scheme_has_multiple_locations?
   end
 
   def set_derived_fields!
+    clear_inapplicable_derived_values
```

Comment:
this should be a bang method for consistency imo

Reply:
agreed, dangerous!